website/docs/r/google_project_organization_policy.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: "google"
+page_title: "Google: google_project_organization_policy"
+sidebar_current: "docs-google-project-organization-policy"
+description: |-
+ Allows management of Organization policies for a Google Project.
+---
+
+# google\_project\_organization\_policy
+
+Allows management of Organization policies for a Google Project. For more information see
+[the official
+documentation](https://cloud.google.com/resource-manager/docs/organization-policy/overview) and
+[API](https://cloud.google.com/resource-manager/reference/rest/v1/projects/setOrgPolicy).
+
+## Example Usage
+
+To set policy with a [boolean constraint](https://cloud.google.com/resource-manager/docs/organization-policy/quickstart-boolean-constraints):
+
+```hcl
+resource "google_project_organization_policy" "serial_port_policy" {
+  project    = "123456789"
+  constraint = "compute.disableSerialPortAccess"
+
+  boolean_policy {
+    enforced = true
+  }
+}
+```
+
+
+To set a policy with a [list contraint](https://cloud.google.com/resource-manager/docs/organization-policy/quickstart-list-constraints):
+
+```hcl
+resource "google_project_organization_policy" "services_policy" {
+  project    = "123456789"
+  constraint = "serviceuser.services"
+
+  list_policy {
+    allow {
+      all = true
+    }
+  }
+}
+```
+
+
+Or to deny some services, use the following instead:
+
+```hcl
+resource "google_project_organization_policy" "services_policy" {
+  project    = "folders/123456789"
+  constraint = "serviceuser.services"
+
+  list_policy {
+    suggested_values = "compute.googleapis.com"
+
+    deny {
+      values = ["cloudresourcemanager.googleapis.com"]
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (Required) The project id of the project to set the policy for. Its format is simply {project_id}.
```

Comment:
I'm not sure the second sentence is really necessary here since we use `project` all over the google terraform provider to mean the exact same thing. Up to you though, I won't block on it.